codegen/templates/chart/operator-deployment.yamltmpl

```diff
@@ -248,7 +248,7 @@ spec:
   ports:
   [[- range $port := $operator.Service.Ports ]]
   - name: [[ lower $port.Name ]]
-    port: {{ $[[ $operatorVar ]].ports.[[ $port.Name ]] }}
+    port: {{ index $[[ $operatorVar ]] "ports" "[[ $port.Name ]]" }}
```

Comment:
Note: This was missed for the extra service definition below for sidecar services